webcam.go

```diff
@@ -34,15 +34,25 @@ type Control struct {
 // Open a webcam with a given path
 // Checks if device is a v4l2 device and if it is
 // capable to stream video
-func Open(path string) (*Webcam, error) {
+func Open(path string) (w *Webcam, err error) {
+	w = nil
```

Comment:
suggestion: how about `var success bool` and the defer checks `!success` and set it to true at the end of the function. might make this less error prone

Reply:
I am fine with that as well. Changed have been made!